src/tools/sssctl/sssctl_logs.c

```diff
@@ -545,15 +544,15 @@ errno_t sssctl_debug_level(struct sss_cmdline *cmdline,
     struct debuglevel_tool_ctx *ctx = NULL;
     struct poptOption long_options[] = {
         {"domain", '\0', POPT_ARG_ARGV, &pc_domains,
-            0, _("Target a specific domain"), NULL},
-        POPT_SERV_OPTION(SSSD, pc_services),
-        POPT_SERV_OPTION(NSS, pc_services),
-        POPT_SERV_OPTION(PAM, pc_services),
-        POPT_SERV_OPTION(SUDO, pc_services),
-        POPT_SERV_OPTION(AUTOFS, pc_services),
-        POPT_SERV_OPTION(SSH, pc_services),
-        POPT_SERV_OPTION(PAC, pc_services),
-        POPT_SERV_OPTION(IFP, pc_services),
+            0, _("Target a specific domain"), _("domain")},
```

Comment:
Imo, 'NULL -> _("domain")' fix could go into dedicated commit as it doesn't fit commit description.

Reply:
Right.